purchase_requisition_auto_rfq/models/purchase_requisition.py

```diff
@@ -61,3 +70,43 @@ def _create_purchase_order(self, seller_id):
         po = self.env["purchase.order"].with_context(**ctx).create(vals)
         po._onchange_requisition_id()
         return po
+
+    def action_all_rfq_related_send(self):
+        self.ensure_one()
+        rfqs = self.env["purchase.order"].search([("requisition_id", "=", self.id)])
+        ir_model_data = self.env["ir.model.data"]
+        try:
+            template_id = ir_model_data.get_object_reference(
+                "purchase", "email_template_edi_purchase"
+            )[1]
+        except ValueError:
+            template_id = False
```

Comment:
This is basically the same:

```suggestion
        template_id = self.env.ref("purchase.email_template_edi_purchase", False).id
```